Tasks/AzureTestPlanV0/manualTests.ts

```diff
@@ -0,0 +1,6 @@
+import tl = require('azure-pipelines-task-lib/task');
+
+export function manualTestsFlow() {
```

Comment:
Should we hide manualFlow from the UI since it is not implmented?

Reply:
Next PR will take care of manual tests flow, we don't need to hide it as of now since the task will not be live right away.